ignite/cmd/network.go

```diff
@@ -42,9 +42,56 @@ func NewNetwork() *cobra.Command {
 	c := &cobra.Command{
 		Use:     "network [command]",
 		Aliases: []string{"n"},
-		Short:   "Launch a blockchain network in production",
-		Args:    cobra.ExactArgs(1),
-		Hidden:  true,
+		Short:   "Launch a blockchain in production",
+		Long: `
+Ignite Network commands are for launching sovereign Cosmos blockchains.
+
+To launch a Cosmos blockchain you need someone to be a coordinator and others to
+be validators. These are just roles, anyone can be a coordinator or a validator.
+A coordinator publishes information about a chain to be launched on the Ignite
+blockchain, approves validator requests and coordinates the launch. Validators
+send requests to join a chain and start their nodes when a blockchain is ready
+for launch.
+
+To publish the information about your chain as a coordinator run the following
+command (the URL should point to a repository with a Cosmos SDK chain):
+
+  ignite network chain publish github.com/ignite/example
```

Comment:
As mentioned I would rather call it `github.com/ignite/planet` as a official project placeholder name for documentation